pymbar/mbar.py

```diff
@@ -620,13 +620,18 @@ def computeExpectationsInner(self, A_n, u_ln, state_map,
         N_k[0:K] = self.N_k
         f_k[0:K] = self.f_k
 
+        # Pre-calculate some quantities outside the for loops
```

Comment:
Can you be more specific what "some quantities" are?  It would useful for people learning the methods exactly what is calculated where at the equation level is.